modules/migration-about-configuring-proxies.adoc

```diff
@@ -132,7 +132,26 @@ spec:
     type: Deny
 ----
 
-[id="configuring-supplemental-groups-for-rsync-pods{context}"]
+[id="choosing-alternate-endpoints-for-data-transfer_{context}"]
+=== Choosing alternate endpoints for data transfer
+
+By default, DVM uses OpenShift Route as an endpoint to transfer PV data to destination clusters. You can choose another type of supported endpoint, if cluster topologies allow.
```

Comment:
How about this for the first sentence: 
By default, DVM uses the endpoint OpenShift Route to transfer PV data to destination clusters.
Second sentence:
If cluster technologies allow, then you can choose another endpoint type.

Reply:
Either looks good to me. Honestly, I like the direct address using second person at the start of the sentence, it fits with the style of the rest of the documentation.